contracts/scripts/util/Rollup.json

```diff
@@ -0,0 +1,1748 @@
+{
```

Comment:
why commit this? you should directly fetch it from `contracts/abi` folder

Reply:
For fetching from contracts/abi we need to `npx hardhat deploy --network localhost` SO to avoid it I added abi to util

Reply:
`npx hardhat compile` will generate abi